client/src/components/Searchfield.js

```diff
@@ -0,0 +1,18 @@
+import "./Searchfield.css";
+import { ReactComponent as SearchIcon } from "../svg/icon-search.svg";
+
+export default function Searchfield() {

```

Comment:
Impressive component and showing css skills! Very good!

Reply:
Thank you :)